### PR TITLE
Added jumpTo function and popover functionality

### PR DESCRIPTION
**Pull request by mnguyen-cayen**
Added a new function called jumpTo to allow for navigating to a month and year directly. For my purposes, I have a month and year dropdown and did not want to use the next and prev buttons.

Also added popover functionality to allow for the use of popovers on the events instead of tooltips. I needed more details to be displayed in the tooltip of the event on the month view so I decided to use a popover.

### CODE REVIEW

File: js/calendar.js [hunk 1]
```diff
@@ -439,16 +441,7 @@ if(!String.prototype.formatNum) {
 
 		data.events = this.getEventsBetween(start, end);
 
-		switch(this.options.view) {
```
Comment (mnguyen-cayen) [inline js/calendar.js]: Can ignore this change. I made it because I wanted to display start and end hours of events on the event-list

Comment (Serhioromano) [inline js/calendar.js]: You mean I have to ignore it? If so I think you have to correct it and add new commits. I mean it have to be done from your side.

**Comment by mnguyen-cayen** [comment]
Sorry, about that. I'm actually pretty new to github. I rolled back those changes. Let me know what you think. Thanks.

**Comment by Serhioromano** [comment]
Please optimize your `switch` I think it should be

```js
switch(this.options.view) {
 	case 'week':
 	case 'day':
 		this._calculate_hour_minutes(data);
 		break;
 }
```

And also I see you define `jumpTo` method but i do not find where you call it. Or you call it outside in your project?

**Pull request closed without merging**